changelog.d/8929.misc

```diff
@@ -0,0 +1 @@
+Automatically drop stale extremities under some specific conditions.
```

Comment:
Is this really a misc change?

Reply:
We tend to put perf related stuff in misc :shrug: